app/components/help/form_component.html.erb

```diff
@@ -0,0 +1,79 @@
+<div class="modal fade" id="contactUsModal" tabindex="-1" aria-labelledby="content-type-prompt" aria-hidden="true">
+  <div class="modal-dialog modal-dialog-centered ">
+    <div class="modal-content">
+      <div class="modal-header">
+        <h3 class="modal-title">Help</h3>
+        <button type="button" class="btn-close" data-dismiss="modal" aria-label="Close"></button>
+      </div>
+      <div class="modal-body" data-controller="help">
+        <%= form_with(url: "#{help_path}.json",
```

Comment:
this shouldn't need `.json`, if you get rid of the responds_to block in the controller.

Reply:
Done